python/create_client/create_utils.py

```diff
@@ -91,6 +91,7 @@ def launch_shotgun_create(sg_connection=None):
             "SHOTGUN_SITE",
             "SHOTGUN_ENTITY_TYPE",
             "SHOTGUN_ENTITY_ID",
+            "PYTHONPATH",  # This fixes issues with Python 3 and Create not launching on Windows.
```

Comment:
Can you add PYTHONHOME too?

Reply:
I’m actually going to revert that fix and make a change to the tk-photoshopcc engine instead, since ideally we want to continue passing the PYTHONPATH env var, since our users might have custom stuff in there. 
The issue is that we for some reason are adding all the Python interpreter paths to the PYTHONPATH before launching Photoshop, which is not ideal, and removing the code doing that appears to have no negative impact since the paths are still included in the sys.path in the launched instance.
Do we still need to do PYTHONHOME?

Reply:
PR here:
https://github.com/shotgunsoftware/tk-photoshopcc/pull/74